src/compiler/crystal/tools/init/template/readme.md.ecr

```diff
@@ -6,11 +6,13 @@ TODO: Write a description here
 
 <%- if config.skeleton_type == "lib" -%>
 1. Add the dependency to your `shard.yml`:
-```yaml
-dependencies:
-  <%= config.name %>:
-    github: <%= config.github_name %>/<%= config.name %>
-```
+
+    ```yaml
```

Comment:
Can there be 3 spaces instead of 4 before every line? Works on GitHub and would be better aligned to the line above.
Currently it's not very good aligned to the "Add" word above by one space as you can see here:
```
1. Add the dependency to your `shard.yml`:

    ```yaml
   ^
```

Reply:
Yes. It's working. Should I change alignment?

Reply:
Yes, that would be good.